posts/2022-01-31-multi-party-data-sharing-with-control.md

```diff
@@ -0,0 +1,271 @@
+
+<figure>
+  <img src="https://user-images.githubusercontent.com/85032783/151488926-9a5e17fc-0ae0-4687-816c-3c70724c6bf3.png" />
+  <figcaption><b>Figure 3</b> - <i>Uni Controls in the Vendia Share Control Plane
+</i></figcaption>
+</figure>
+
+We can slightly modify the RBAC terms from the previous section to make them specific to Uni management.
+
+#### Key Terminology
+* **Action** - The right to call one of the Uni management operations exposed by the Vendia Share control plane
+* **Resource** - The Vendia Share Uni or Unis against which a Uni management Action can be performed
+* **Capability** - A mapping of Action to Resource, linking the right to call (an operation) with the right to manage (a Uni)
+* **Role** - A named collection of Capabilities; used to determine if a Uni management or data operation, executed by invoking the Vendia Share control plane or data plane API, is permitted or denied
+
+There are a set of [Actions](https://www.vendia.net/docs/share/rbac#user-actions) relevant to Uni management.  You'll notice these are a subset of the `ActionEnum` defined in the Vendia Share [control plane API](https://www.vendia.net/docs/share/share-graphql-schema#vendia-share-graphql-schema-reference).  Further, you'll notice that each Action defined corresponds to a user-specific query or mutation available through the API.
+
+<table>
+  <tr>
+   <td><strong>Action</strong>
+   </td>
+   <td><strong>GraphQL Query or Mutation Operation</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>UNI_CREATE
+   </td>
+   <td>register
+   </td>
+  </tr>
+  <tr>
+   <td>UNI_DELETE
+   </td>
+   <td>destroy
+   </td>
+  </tr>
+  <tr>
+   <td>UNI_RESET
+   </td>
+   <td>reset
+   </td>
+  </tr>
+  <tr>
+   <td>UNI_JOIN
+   </td>
+   <td>join
+   </td>
+  </tr>
+  <tr>
+   <td>UNI_INVITE
+   </td>
+   <td>invite
+   </td>
+  </tr>
+  <tr>
+   <td>UNI_DELETE_NODE
+   </td>
+   <td>NA
+   </td>
+  </tr>
+  <tr>
+   <td>UNI_GET
+   </td>
+   <td>getUni, listUnis, getNode
+   </td>
+  </tr>
+  <tr>
+   <td>UNI_EVOLVE_SCHEMA 
+   </td>
+   <td>evolve
+   </td>
+  </tr>
+  <tr>
+   <td>UNI_QUERY
+   </td>
+   <td>NA - future use
+   </td>
+  </tr>
+  <tr>
+   <td>UNI_MUTATE
+   </td>
+   <td>NA - future use
+   </td>
+  </tr>
+</table>
+
+
+There are a different set of [Resource](https://www.vendia.net/docs/share/rbac#uni-formats) formats that are permitted for Uni management purposes.  The formats can be used to describe one or more Unis in a single statement.  These are the Unis against which an Action from above can act upon.
+
+
+### Examples
+We've created a set of examples to demonstrate how Uni Controls can be viewed and managed and the implications of their settings on Uni management operations.  These include invocation of control plane APIs using the [Vendia Share CLI](https://www.vendia.net/docs/share/cli) and a GraphQL client.
```

Comment:
Is there a missing link to examples here?

Reply:
Yes.  All links are missing pending a merge of https://github.com/vendia/examples/pull/27.